diff --git a/viteSvgLoader.ts b/viteSvgLoader.ts
index a119e3ed..50bfbbcc 100644
--- a/viteSvgLoader.ts
+++ b/viteSvgLoader.ts
@@ -9,8 +9,14 @@ export const svgLoader = (): Plugin => ({
     if (id.endsWith('.svg')) {
       const extractedSvg = fs.readFileSync(id, 'utf8');
       const datauri = svgToMiniDataURI.toSrcset(extractedSvg);
-      return `export default "${datauri}"`;
+      return {
+        code: `export default "${datauri}"`,
+        map: null,
+      };
     }
-    return code;
+    return {
+        code: code,
+        map: null,
+    };
   },
 });
